bundles/AdminBundle/Resources/public/js/pimcore/asset/folder.js

```diff
@@ -291,16 +291,22 @@ pimcore.asset.folder = Class.create(pimcore.asset.asset, {
                 });
             }
 
-            var user = pimcore.globalmanager.get("user");
+            let user = pimcore.globalmanager.get("user");
+
             if (user.admin) {
```

Comment:
This is the only place where it is checked if the user is admin. Why is that? I don't think that is necessary so I would remove it if it is ok? Or it should be implemented in the other places as well